python/src/otel/otel_sdk/otel-instrument

```diff
@@ -133,7 +133,7 @@ fi
 #       )
 #   )
 
-export LAMBDA_RESOURCE_ATTRIBUTES="cloud.region=$AWS_REGION,cloud.provider=aws,faas.name=$AWS_LAMBDA_FUNCTION_NAME,faas.version=$AWS_LAMBDA_FUNCTION_VERSION";
+export LAMBDA_RESOURCE_ATTRIBUTES="cloud.region=$AWS_REGION,cloud.provider=aws,faas.name=$AWS_LAMBDA_FUNCTION_NAME,faas.version=$AWS_LAMBDA_FUNCTION_VERSION,faas.instance=$AWS_LAMBDA_LOG_STREAM_NAME";
```

Comment:
The change looks ok, I'm a bit surprised there isn't a better environment variable to use to identify the specific instance.

Reply:
I think this serves the purpose effectively and I'm not aware of any better alternative.  I don't think it's a one-way door as any useful alternative would also provide a unique ID.